apps/ensnode/test/ponder-helpers.spec.ts

```diff
@@ -57,6 +59,28 @@ describe("ponder helpers", () => {
     });
   });
 
+  describe("parseEnsRainbowEndpointUrl", () => {
+    it("should parse a valid ENSRainbow endpoint URL", () => {
+      expect(parseEnsRainbowEndpointUrl("https://api.ensrainbow.io")).toBe(
+        "https://api.ensrainbow.io/",
+      );
+    });
+
+    it("should throw an error if the URL is invalid", () => {
+      expect(() => parseEnsRainbowEndpointUrl("almost_an_URL")).toThrowError(
+        "'almost_an_URL' is not a valid URL",
+      );
```

Comment:
@lightwalker-eth this test case addresses your request about testing invalid URL value provided as `ENSRAINBOW_URL` env var.